utils/docker/cc-builder/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FROM hyperledger/fabric-private-chaincode-dev:latest
+
+RUN cd $FPC_PATH
+RUN make min
+
+WORKDIR $FPC_PATH/demo/chaincode
```

Comment:
I would have imagined this is a generic cc builder where you "inject" the code via volume and then call make (or whatever you used to build the chaincode) in that injected volume. In that case, there wouldn't be any need for WORKDIR?  Seems a bit strange to have this WORKDIR here pointing to demo ...

Reply:
I agree with Michael

Reply:
I suggest, for now we can keep it but add a comment here (maybe also the docker command that it needs to mount some chainode inside this container)